types/application.d.ts

```diff
@@ -72,3 +83,21 @@ export const appLanguage: string;
  * User's OS-wide locale setting. May not match the XD UI, since XD does not support all world languages. Includes both language and region (e.g. "fr_CA" or "en_US").
  */
 export const systemLocale: string;
+
+/**
+ * Information about the document which this instance of the plugin is attached to.
+ *
+ * > **Tip**
+ * >
+ * > _This does **not** indicate the frontmost "active" document window in the XD application._
+ * >
+ * > In XD, each document window loads a separate copy of your plugin. When a given instance of your plugin calls this API, you will always receive information about the document that this instance of the plugin is attached to, even if it's not the active window.
```

Comment:
Formatting. The [source note](https://adobexdplatform.com/plugin-docs/reference/application.html#applicationactivedocument--documentinfo) doesn't have a line break. (A single line-break in Markdown is ignored. Even though this shows up on two separate lines, it is rendered as a single, flowing paragraph.)

```suggestion
 * > _This does **not** indicate the frontmost "active" document window in the XD application._
 * > In XD, each document window loads a separate copy of your plugin. When a given instance of your plugin calls this API, you will always receive information about the document that this instance of the plugin is attached to, even if it's not the active window.
```